src/custom_nodes/east_ocr/nms.hpp

```diff
@@ -0,0 +1,221 @@
+#pragma once
```

Comment:
Add the original license

Reply:
Added license from original github repository to nms.hpp header